[2025-04-29T04:38:35.108+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2025-04-25T00:00:00+00:00 [queued]>
[2025-04-29T04:38:35.119+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2025-04-25T00:00:00+00:00 [queued]>
[2025-04-29T04:38:35.125+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-04-29T04:38:35.125+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-04-29T04:38:35.129+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-04-29T04:38:35.137+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): store_user> on 2025-04-25 00:00:00+00:00
[2025-04-29T04:38:35.143+0000] {standard_task_runner.py:55} INFO - Started process 131 to run task
[2025-04-29T04:38:35.144+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'scheduled__2025-04-25T00:00:00+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpozv64qci']
[2025-04-29T04:38:35.147+0000] {standard_task_runner.py:83} INFO - Job 38: Subtask store_user
[2025-04-29T04:38:35.185+0000] {task_command.py:388} INFO - Running <TaskInstance: user_processing.store_user scheduled__2025-04-25T00:00:00+00:00 [running]> on host b08efc87d46b
[2025-04-29T04:38:35.214+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2025-04-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-04-25T00:00:00+00:00
[2025-04-29T04:38:35.215+0000] {postgres.py:158} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2025-04-29T04:38:35.219+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2025-04-29T04:38:35.221+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-04-29T04:38:35.224+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20250425T000000, start_date=20250429T043835, end_date=20250429T043835
[2025-04-29T04:38:35.242+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-04-29T04:38:35.250+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
